src/index.ts

```diff
@@ -250,6 +250,46 @@ function strEncoder(data: Buffer): string {
   return encodeEd25519PublicKey('ed25519PublicKey', data)
 }
 
+function tezosAddressEncoder(data: Buffer): string {
```

Comment:
It'd be a good idea to link to some documentation on these formats along with the function, so people can verify it's doing the right thing.

Reply:
The only relevant reference I could find was from the stack exchange above. 

https://tezos.gitlab.io/api/p2p.html?highlight=contract_id#contract-id-22-bytes-8-bit-tag

I will add them as comments.